### PR TITLE
Move flyover methods from BridgeSupport to BridgeUtils

### PR DESCRIPTION
**Pull request by julia-zack**

### CODE REVIEW

**Review by apancorb — APPROVED** [review]
LGTM

**Comment by apancorb** [comment]
Is this intended to be used in the context of the svp `fundTx`?

File: rskj-core/src/test/java/co/rsk/peg/PegUtilsTest.java [hunk 1]
```diff
@@ -61,6 +66,34 @@ void init() {
...
+        Script scriptPubKey = getFlyoverScriptPubKey(flyoverDerivationHash, redeemScript); // OP_HASH160 outputScript OP_EQUAL
+        byte[] outputScript = Hex.decode("18fc3b52a5b7d5277f41b9765719b45bfa427730");
+
+        assertArrayEquals(outputScript, scriptPubKey.getPubKeyHash());
+    }
+
+    @ParameterizedTest
+    @MethodSource("derivationHashAndRedeemScriptArgs")
+    void getFlyoverAddress_fromRealValues_shouldReturnSameRealAddress(Keccak256 flyoverDerivationHash, Script redeemScript) {
+        Address flyoverAddress = new Address(btcMainnetParams, "33y8JWrSe4byp3DKmy2Mkyykz2dzP8Lmvn");
```
Comment (marcos-iov) [inline rskj-core/src/test/java/co/rsk/peg/PegUtilsTest.java:81]: Check sonar warning, you can use Address.fromBase58 here

Comment (julia-zack) [reply rskj-core/src/test/java/co/rsk/peg/PegUtilsTest.java]: thanks! updated

**Pull request merged**